process_report/tests/unit_tests.py

```diff
@@ -473,9 +476,9 @@ def setUp(self):
             )
             .astype(
                 {
-                    "Initial Credits": pandas.ArrowDtype(pyarrow.decimal128(21, 2)),
-                    "1st Month Used": pandas.ArrowDtype(pyarrow.decimal128(21, 2)),
-                    "2nd Month Used": pandas.ArrowDtype(pyarrow.decimal128(21, 2)),
+                    "Initial Credits": object,
```

Comment:
Why is this an object now?

Reply:
The reason I made the change is very silly and was a oversight when refactoring the test cases. This boiled down to me no longer letting the test case [type cast the output old_pi_df](https://github.com/CCI-MOC/process_csv_report/blob/287afea5875b32bf25b8727b2790c54f14137328/process_report/tests/unit_tests.py#L560-L566). I have reverted this change, so the diff should be simpler now.